src/DssSpell.sol

```diff
@@ -18,14 +18,33 @@ pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "lib/dss-interfaces/src/dss/GemJoinAbstract.sol";
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+
+interface Initializable {
+    function init(bytes32) external;
+}
+
+interface Hopeable {
+    function hope(address) external;
+}
+
+interface Kissable {
+    function kiss(address) external;
+}
+
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory,address,uint48,uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
 
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d04bee8084c5724496ef72a0c06d00e2c7b8905e/governance/votes/Executive%20vote%20-%20April%202%2C%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/c9dd499d90ec08ec55fa242456045bb712932006/governance/votes/Executive%20vote%20-%20April%209%2C%202021.md -q -O - 2>/dev/null)"
     string public constant description =
-        "2021-04-02 MakerDAO Executive Spell | Hash: 0xdb1339f86f456759bc4ec97f49b7b10abb86d97bbb9f56577731efd715d978c9";
+        "2021-04-09 MakerDAO Executive Spell | Hash: 0x6362b56cd687d047ecaa6b4de28d84703e488a2dc71f45557a589dbac95736f2";
```

Comment:
The `description` probably needs to be updated, including the following commit changes with the updated spell date:
https://github.com/makerdao/community/commit/2a7a8c915695b7298fe725ee3dc6c613fa9d9bbe